src/rendering/SceneManager.cc

```diff
@@ -128,6 +128,10 @@ class gz::sim::SceneManagerPrivate
   /// also sets the time point in which the animation should be played
   public: AnimationUpdateData ActorTrajectoryAt(
       Entity _id, const std::chrono::steady_clock::duration &_time) const;
+
+  /// \brief Holds the spherical coordinates from the world.
+  public: math::SphericalCoordinates sphericalCoordinates =
+          math::SphericalCoordinates();
```

Comment:
The default constructor should already be called, so I think you can skip the `=`.

Reply:
Done https://github.com/gazebosim/gz-sim/pull/1556/commits/c5ea0cb0204a46199f2c5be54da3361fb65f7b81